index.bs

```diff
@@ -736,10 +712,13 @@ To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
      1. Let |transport| be [=this=].
      1. If |transport|'s [=[[State]]=] is `"closed"` or `"failed"`,
         then abort these steps.
-     1. Set |transport|'s [=[[State]]=] to `"closed"`.
      1. Let |session| be |transport|'s [=[[Session]]=].
      1. [=session/Terminate=] |session| with |closeInfo|.{{WebTransportCloseInfo/errorCode}}.
 
+       Note: This also [=resets=] or [=sends STOP_SENDING=] [=WebTransport streams=] contained in
+       |transport|'s [=[[SendStreams]]=] and [=[[ReceiveStreams]]=].
```

Comment:
Does it? A note implies this is done elsewhere, yet I see nothing about this in [section 5](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-5). Unless there's a different section we can reference, this note should probably morph into normative steps that take care of this here (in parallel) I think.

Reply:
"the endpoint MUST stop sending new datagrams and reset all of the streams associated with the session." It was not clear to me whether stream reset was included in the session termination action, so I added this note. @vasilvv do you have any comments?